File: shared/src/main/java/xyz/klinker/messenger/shared/util/RedirectToMyAccount.java
package xyz.klinker.messenger.shared.util;

import android.content.Intent;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;

import xyz.klinker.messenger.shared.MessengerActivityExtras;

public class RedirectToMyAccount extends AppCompatActivity {

    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        overridePendingTransition(0,0);

        Intent messengerActivity = ActivityUtils.buildForComponent(ActivityUtils.MESSENGER_ACTIVITY);
        messengerActivity.putExtra(MessengerActivityExtras.INSTANCE.getEXTRA_START_MY_ACCOUNT(), true);
        startActivity(messengerActivity);

        overridePendingTransition(0,0);
        finish();
    }
}

File: shared/src/main/java/xyz/klinker/messenger/shared/util/RedirectToMyAccount.kt
package xyz.klinker.messenger.shared.util

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity

import xyz.klinker.messenger.shared.MessengerActivityExtras

class RedirectToMyAccount : AppCompatActivity() {

    public override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        overridePendingTransition(0, 0)

        val messengerActivity = ActivityUtils.buildForComponent(ActivityUtils.MESSENGER_ACTIVITY)
        messengerActivity.putExtra(MessengerActivityExtras.EXTRA_START_MY_ACCOUNT, true)
        startActivity(messengerActivity)

        overridePendingTransition(0, 0)
        finish()
    }
}
